### PR TITLE
[Arista] Add missing configs for 720DT-48S

### PR DESCRIPTION
**Pull request by andywongarista**
#### Why I did it
Followup to https://github.com/Azure/sonic-buildimage/pull/10656. This change adds the remaining configs for the 720DT-48S platform.

#### How I did it
Adds the following:
- gearbox_config.json and other gearbox-related config files, to enable traffic on external PHY ports (Ethernet0-23)
- sensors.conf
- pcie.yaml
Also add missing facts in platform.json

#### How to verify it
- `show interfaces status` shows links up on interfaces Ethernet0-23
- traffic flows with no errors on interfaces Ethernet0-23
Note: above testing depends on https://github.com/Azure/sonic-buildimage/pull/11154 and https://github.com/Azure/sonic-swss/pull/2367, as well as having the appropriate PAI driver.

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [x] 202205

### CODE REVIEW

**Review by Blueve — CHANGES_REQUESTED** [review]
Hi @andywongarista, can you update the alias column with etpN naming (N start from 1) ?

https://github.com/andywongarista/sonic-buildimage/blob/master/device/arista/x86_64-arista_720dt_48s/Arista-720DT-48S/port_config.ini 

@lguohan I think we need this back port to 202205?

File: device/arista/x86_64-arista_720dt_48s/td3x2-a720dt-48s-flex.config.bcm [hunk 1]
```diff
@@ -219,30 +219,30 @@ portmap_56.0=64:10
...
-port_init_autoneg_16.0=0
-port_init_autoneg_17.0=0
-port_init_autoneg_18.0=0
-port_init_autoneg_19.0=0
-port_init_autoneg_20.0=0
-port_init_autoneg_21.0=0
-port_init_autoneg_22.0=0
-port_init_autoneg_23.0=0
-port_init_autoneg_24.0=0
+port_init_autoneg_1.0=1
```
Comment (Blueve) [inline device/arista/x86_64-arista_720dt_48s/td3x2-a720dt-48s-flex.config.bcm:222]: Do we need to set the autoneg to `true` in phy{N}_config.json?

**Comment by lguohan** [comment]
@blueve, they can do it in another pr.

File: device/arista/x86_64-arista_720dt_48s/td3x2-a720dt-48s-flex.config.bcm (hunk 1 above)
Comment (andywongarista) [reply device/arista/x86_64-arista_720dt_48s/td3x2-a720dt-48s-flex.config.bcm:222]: From testing the PAI driver it seems that setting the autoneg in the phy*_config.json files is not necessary.

**Comment by andywongarista** [comment]
> Hi Andy, please update the port config ini in another PR

Sure

**Review by Blueve — APPROVED** [review]

**Comment by Blueve** [comment]
@andywongarista @jimmyzhai are we good to merge this PR?

**Comment by andywongarista** [comment]
> @andywongarista @jimmyzhai are we good to merge this PR?

Yes

**Comment by yxieca** [comment]
@andywongarista  please raise separate PR for 202205 branch.

**Comment by Blueve** [comment]
@andywongarista I installed image:
```
SONiC Software Version: SONiC.master.128992-cdd278611
Distribution: Debian 11.4
Kernel: 5.10.0-12-2-amd64
Build commit: cdd278611
Build date: Sun Jul 31 16:35:13 UTC 2022
Built by: AzDevOps@sonic-build-workers-001UZN
```

But `show int status` still not list any port. Do you have any suggestion?

**Pull request merged**